### DISCUSSION TITLE
FileNotFoundError: Could not find module 'C:\Users\Anaconda3\Lib\site-packages\torch_sparse\_convert_cuda.pyd' (or one of its dependencies). Try using the full path with constructor syntax.

### DISCUSSION THREAD

**Discussion by bebenabu** [pyg-team/pytorch_geometric#4939] [Q&A]
Hello,
I'm learning pytorch_geometric using the examples but having issues with torch_sparse. It seems to be correctly installed but there is a path issue.
The error message is talking about cuda while I'm working on CPU in an Anaconda environment.
Did anyone get this kind of issues ?
![image](https://user-images.githubusercontent.com/36326507/177784283-7bb97e45-7903-4e8b-8713-01fbc37bfcfd.png)

**Comment by rusty1s** [comment]
How did you install `torch-sparse`? What's the output of `torch.__version__`?
